AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_processor_batch_tests.py

```diff
@@ -330,41 +326,42 @@ def test_TwoAssetsWithSameProductName_ShouldProcessAfterRename(self, asset_proce
         """
 
         def run_ap_debug_skip_atom_output(asset_processor):
-            result, output = asset_processor.batch_process(capture_output=True, extra_params=["--debugOutput",
-                                                                                              "--regset=\"/O3DE/SceneAPI/AssetImporter/SkipAtomOutput=true\""])
+            result, output = asset_processor.batch_process(capture_output=True, extra_params=["--regset=\"/O3DE/SceneAPI/AssetImporter/SkipAtomOutput=true\""])
```

Comment:
This nested function's name suggests that `--debugOutput` was an important param. Appears like an unintentional change, but the AP maintainers should be able to clarify whether the change is appropriate. Should probably remove debug from the name if it's no longer used.

Reply:
Yeah debug wasn't needed in this case. I'll remove it.
I think this was actually duplicate code from another module, which needed both, it would likely be best to add the skip atom output either to one of the assetpipeline fixtures or to Tools/LyTestTools/ly_test_tools/o3de/AssetProcessor.py